manifests/database.pp

```diff
@@ -14,6 +14,7 @@
   }
 
   pulpcore::admin { 'migrate --noinput':
+    timeout     => 0,
```

Comment:
I assume we should adopt the same strategy as puppet-foreman here? Add a parameter to this class just in case, and default to 3600.

Reply:
yepp, updated to match the way we discussed for puppet-foreman.